README.md

```diff
@@ -80,8 +78,7 @@ config-version: 2
 
 vars:
     iterable_source:
-        campaign_suppression_list_history: "{{ source('iterable', 'campaign_supression_list_history') }}" 
-        using_old_spelling: true # default false
+        campaign_suppression_list_history_table: "campaign_supression_list_history"
```

Comment:
I think it would be best if we can be consistent with the variable naming for this package. My thoughts would be to change the name to:
`iterable__campaign_suppression_list_history_table`

I know it is a mouthful, but I feel it fits our current naming convention more.

Reply:
ah yep totally agree